internal/code-generator/assets/languages.go.tmpl

```diff
@@ -0,0 +1,9 @@
+package language
+
+type Language string
+
+const (
```

Comment:
As those will end up being a part of the API surface of `data` package - it would be nice to document them as well e.g by moving here part of the PR description explaining how it will be used, etc.